tumblebit/__init__.py

```diff
@@ -24,12 +25,18 @@
 ########################################################
 
 # Change path to where libressl library is
-# TODO: Add an option to specify the path in some sort of
-#       config file.
-if(system() == "Darwin"):
-    path = "/usr/local/opt/libressl/lib/libssl.dylib"
+try:
+    config.LIBRESSL_PATH
```

Comment:
python noob here but why not `path = config.LIBRESSL_PATH` and removing the else at the end ?